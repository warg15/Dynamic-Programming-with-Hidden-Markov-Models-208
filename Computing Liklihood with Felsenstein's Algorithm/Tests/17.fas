>Pigeon
AGTAAGAATCTCAAAAGACGTGCGAAGACATAGTCACGAGTGATACCATCAGTATGTCAACTCATACCCGTAATAAGATAACAATACTACATGATCTATTACAGATAGAGATATAGAAGAACAAATAATCTAAGTCCCAACATTATCGACAGGATAGGTGGTATGATGCCATCTTCCGACAAGAAGAGTCGTACGTGAGC
>Yellow-throated-sandgrouse
AGTAAGCATCTCGAAAAACGTGCGTAGGCATAGACACGAGTGATACCATCAATATGTCAATTCATACTCGTAGCAAGATAGCAAGACCACATAACCTATTACCGATAAAGCTACGGAAGAATAAATAATCTAAGTCCCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAGGAAGTGTCGAACGTGAGC
>Bald-eagle
AGTAAGCATCTCGAAACACGTACATGGGCATAGACACGAGTGATACCATAAATATGTCAACTTATACTCGGAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAACTAATCTAAGTCCCAACATTATCACCAGGTTAGATGGTATGGTGCTGTCTTCCGACAAGAAGTGTCGAACGTGAGC
>Great-cormorant
AGTAAGGATCTCGAAAAACATGCATAGGCATAGACACGTGTGATAACAACAATATGTGAACCTATACTCGTAGCAAGCTAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATATATAATCTAAGTTTAAACACTATTACGAAGCTAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCGAACGTGACT
>Rifleman
AGTAAGCAGCTCGGACAACGTGCTTAGGCATAGGCACGCGTGATACCATCAATATGTAAACACATACCCGCAGCGAAATAGCAACGCTACATGAACTATTACCGATAGAGATATGGAAGAATAAATAATCCAAGCCCCAGCATTATCACCGGGATAGGTGATATGGGGCCGTCTTCCGACAATACGTGTCGAACGTGAGC
>Golden-collared-manakin
AGTAAACATCTCGAAAAACGTGATTAGGCATAGACACGCGTGGTACCATCAATATGTTAACTCATACCCGTAGCAAGATAACAGTACTATATGAACTATTACCGATAGAGGTATGGAAGAATAAATAATCTAAGTCCCAACATTATCATCAGGATAGGTGGTACGGGGCCGTCTGCCGACAATGAGCGTCGGACGTAAGC
>Downy-woodpecker
AGAAAACATCCCGAAAAACGTGCGTAGGCATAGAGGCAAGTGATACCAGCAAGATGCCAACTCCTACTAGTACCAGAAAAACAATACTATATGACCTATCAGCGGGAGGAATATGTAAGCATAAATAATCTAACTTCCAACACTATCCCCAAGATAGATGGTATAGTGCCGCCTTCCGAAAAGAAGTGTCGAACATGGGC
>American-flamingo
AGTAAGCATCTCGAAAAACATGCGTAGGCATAGACACGAGTGATATCATCAATATGTCAACTCATACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAAATAATCAAAGTCCCAACATTATCACCAGGATAGGAGGTATGGTGCCGTCTTCCGATAAGAAGTGTCGGACGTGAGC
>American-crow
AGTAAACATCTCGAAAAACGTGCTTAGGCATAGACACGCGTGGTACCGTCAATATGTCAAGTCATACCCGTAGCGAGATAGCAACACTATATGAACTATTACCAATAATGGTATGAGAGAATAAATAACCTAAGTCCTAACATTATCACCAGGACAGGTGGTATGGGGCCGTTTTACGACAATAAGTGTTGAACCTGAGC
>Adelie-penguin
AGTAAGCATCTCGAGAAACGTGCGTAGGCACCGACACGCGTGATACCATCAATATGTCAACTCGTACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCCCAACATCATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCGAACATGAGT
>Dalmatian-pelican
AGTAAGCACCTCGAAAAACGTGCGTAGGCATAGACACGCGTGATACCATCAATATGTCAACTCTTACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAGGAATAAATAATTTAAGTCCCAACATTATCACCAGGATAGGTGGTATGGTGCCATCTTCCGAGAAGAAGTGTCGAACGTGAGT
>Hoatzin
AGTAGGCATCTCGAAAAACGTGCGTAGACATAGACACGAGTGATATCACCAATATGTCAATTCATACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCCCAACATTATCATCGGGATAGGTGGTATTGTGTCGTCTTCCGATAAGAAGTGTCGAACGTAGGC
>Speckled-mousebird
AGTAAGCATCTCGAAAAGCGTGCGTAGGCATGGTCGCGAGTGATGCCATCAATATGTTAACTCAGACTCATAGCAAGATAACACCACTACATGACCTATTACCGATAGAAATATGGAAGAATAGATAATCTAAGTCCCAACTTTATCACTAGAATAGGTTGTAGGGTGCCGTCTTCCGCACAGAAGTGTCGAACGCGAGC
>Sunbittern
AGTAAGCATCTCGAAAAACGTGCGGAGGCATAGACACGAGAGATTCCACTAGTATACCAACTCATCCTCGTAGCAAGATGACAATACTACATAACCTATTACCGATAGGGATATGGAAGAATAAATAATCTAAGTCCCAACATGATCACCAGGATATGTGGTATGTTGCCGTCTTCCGACAAGAAGCGTCGAACGCAAGC
>Red-legged-seriema
AGTAAGCACCTCGAAAAACGTGCGTAGGCATAGACACGAGTGATACCATCAGTATATTAACTCACACTCGTATCAAGATAACAATACTACATGACCTATTACCGACAGAGATATGGAAGAATAAATAATCTAAGTCCCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGTCAAGAAGTGTCGAACGTGAAC
>Pekin-duck
CGTAAGCATCTCAAAAAACGTGCGTAGGCAGAGCCATGAGTGATACCATTAATGTGTCAACTCATGCTTGTAGCAAGATGACCATACTACTTGATTTATTACCGACAAAGATATGAAAGAATAGATGATGTAAGCCCCAACATTATTACCAGGATAGGTGGTATAGTGCCGTCTTCCGAGAAGAAGTGCCGAACAGGAGC
>Medium-ground-finch
AGTAAACATCTCGAAAAACGTGCTTAGGTATAGACACGCATGGTACCGTCAATTTGTCAACTCATACCCATAGCGAGATAACAATACTATATGAACTATTACCAATAGTGGTATGAAAGAATAAATAACCTAAGTCCCAACATTATCGCTAGGATAGCTGGTATGGGGCCGTTTTCCGACAATAGGGGTCGAACCTGAAC
>White-throated-tinamou
AGTCAGCCTCTTGAGAAGCATGCGTAGGCGTAGGCCCAAGTGATAGCATGAACATGTCAACTTAGGCTTGTATCAACATAACAACACTACCTGAACTATTTCCGATACAAAGCGGAAAGAATGAATGATGGAAGTCCCAAAAGGAATACCAGGATAGGGAGTATGGTGCCGTCTTCCGACAAGGAGTTTTGAGCGTGAGT
>Common-ostrich
AGTAAGCTTCTCGAGAAGCATGCGTAGGCATGGGCCTAAGTGATGTCATGAATATGTCAACTTGTATTTGTAGTAAGATAACAACACTACCTGACCTATTACCGATAAAGAGAGGGAAGAATAAATGTTGTAAGCCCCAAAATGATTACCAGGATAGGGGGTATGGTGCCGTCTTCCGAGAAGGAGTGTCGAGCGTGAGC
>Crested-ibis
AGTAAGCATCTCGAAAAACGTGCGTAGGCATAAACACACGTGATACCATCAATGTGTCGACTCATACTCGTAGCAAGATAACAATACTACATGACGTATTACCGATAGAGATATGAAAGAATAAATAATCTAAGTTCCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCGAACATGAGT
>Northern-fulmar
AGTAAGCATCTCGAAAAACGTGCGTAAGCATAGACACGCGTTATACCATCAATATGCCAACTCATACTCGTAGCAAGATAACAATACTACATGATCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCCCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAACAAGTGTCGAACGTGAGT
>Bar-tailed-trogon
AGTAAGTATCTCGAAAAACGTGCGTAGACATAGACACTAGTGGGACTATCAGTATGTCAACTCATACTCGTAGCAAGATAACAATACTACATGACCTATTCCCGATAGAGATATGGAAGGATAAAGAATCTAAGTCCCAACATTCTCACCAGGATAGGTGGTATGGTGCTGTCCTGCGACAGGAAGTGTCGAACGTAAGC
>Carmine-bee-eater
AGTAGGCATCCCGAAAAACGTGCGTAGGCATGGTCACAAGTGATACCATCAATATGTCAACTCATACTCCTAGCTAGATAACAATACTGCATGATCTATTACCGATAGAGATATGGGCGAACAAATAATCTAAGACCCGACATTATCGCCAGGATAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCAAACGTGAGC
>Barn-owl
AGTAAGCATTTCGAAAAAAGTGCGTAGGCATAGACACGAGTGATACCATCAATATGTCAACTCACACTCGTAGCAGGATAACAATACTACATGACCTATGACCGATAGAGATATGGAAGAAAAAGTAATCTAAGTCTCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCGGACAAGAAGTGTCGATCGTGAGC
>Kea
AGTAGGCATATCGAAAAACGTACTTAGGCATAGACACGAGTGATACCACCCACATATCAACTCATATTCGTAGTAAGATAACAATACTAAATGACCTATTACCGATAGAAATATGGAAGAATAAATAATCTAAGTCCCAACATTATTACCGGGATAGGTGGTATGGTGCCGTCTTCCGACAAAAAGTGACGAACGTCAGC
>Emperor-penguin
ACTAAGCATCTCGAGAAACGTGCGTAGGCACAGACACGCGTGATACCATCAATGTGTCAACTCATACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCCCAACACCATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCGAACGTGAGT
>White-tailed-eagle
AGTAAGCATCTCGAAACACGTACATGGGCATAGACACGAGTGATACCATAAATATGTCAACTTATACTCGGAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAACTAATCTAAGTCCCAACATTATCACCAGGTTAGATGGTATGGTGCTGTCTTCTGACAAGAAGTGTCGAACGTGAGC
>Little-egret
AGTAGGCATCTCGTAAAACGTGCGTAGGCATAGACACGTGTGATACCATCAATCTATCAACTCATACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCGCAACATGATCTCCAGGATAGATGGTATGGTGCCGTTTACCGACAAGAAGTGTCGAACGTGAGA
>Rhinoceros-hornbill
AGTAAACATCCCGAAAAACGTGCGTAGGTACAGACGCGAGTGATAGCATGAATATGTCAACTCAGACACGTAGCAAGATAACAATACTACATGACCTATTACCGTTAGAGATATGGGAGAATAAAGAACCTAAGGTCCAACATTATCAACAGGATTGGTGGTCTGATGCCGTCTTCCGACAAGAAGGGTCGGACGTGAGT
>White-tailed-tropicbird
AGTAAGCATCTCGAAAAACATGCGTAGGCATAGACACGAGTGATACCACCAATATGTCAACTCATACTCGTAGCACGATAACAATACTACATAACCTATTACCGATAGAGATCTGGACGAATAAATAATCTAAGATCCAACATTATCACCAGGACAGGTAGTATCGTGCCGTCTTCCGACAAGAAGTGTCGAACGTGAAC
>Peregrine-falcon
AGTAAGCATCTCGAAAAACGTGCGTAGGCATAGACGCGAGTGATACCAGCAATATGCCAACTCATTTTCGTAACAAGCTAACAATACTACATAATCTATTACCGACAGAGATATGGAAAAATAATTAATCTAAGTCGCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGCCAAGGAGTGCCGAAGGTGAAC
>MacQueens-bustard
AGTAAGCATTTCGAAAAACGTGCGTAGGCATAGACACGAGACATACCACCAATATATCAACTCATACTCGTAACAAGACAGCAATACTACATAACCTATTACCGATAAAGATATGAAAGAATAACTAATCTAAGTCTCAACATTATCACCAGGATAGGTGGTATGGTGCCGTCTTCCGACAAGAAGTGTCGAACGTGAGC
>Budgerigar
AGTAGGCATCTCGAAAAATGTACTTAGGTATAGACACGAGTGATACCCCCAATATATAAACTCATATTCGTAGTAAGATAGCAATACTACATGACCTATTACCGATAGAAATATGGAAGAATGAGTAATCTAAGTACCAACATTATTAGCAGGATAGGTGGTATGGTTCCGTCTTCTGACAAAAAGTGACTAACGTCAAC
>Red-crested-turaco
AGTAAGCATCTCAAAAAACGTGCGTAGGCATAGACACGAGTGATACCATCAATATGTAAACTCGTACTCGTAGCAAGATAACAATACTACGTGACCTATTACCGATAGAGATGTGAAAGAATAAATAATCTAAGTCCAAACATTATCACCAGTATAGGTGGTATGGTGCTGTCTTCCGACAAGAAGTGTCGAACGTCAGC
>Brown-mesite
AGTAAACATCTCGAAAAACGTGCGTAGACATAGACACGAGTGATACCAGCAATATGTCAATTCATACTCGTACCAGGATAACAATACTACATGACCTATCACCGATAGCGATATGGAAGAATAAATAATTTCAGTCCCAATATTATCACCAGGATTCGTGGTATAGTGCCGTCTTTCGACGAGAAGTGTCGAACGTGCGC
>Chicken
CGGAAACATCTCGAAGAACCCGCGTAGGCATAGGAATGGGCGATACCATCAATATGTCAACTCATACTTGTAGCGAGAAAAACATGCTACTTCATTTATTGCAGGTAAAAATATGGGAGAATAAACAATCTAAGTCCCAACATTATTACCAGGATAGGTGGTATGGTGCCGTATTCCGACAAGAAATGTCGAATTTGATT
>Red-throated-loon
AGTAAGCATCTCGAAACACGCGCATAGGCATAGACACGCATGATACCATCAATATGTCAACTCATATTCATAGCAAGATAACAATACTACATAACCTATTACCGATAAAGATTTGGAAGAATAAATAATCTAGGTCTCAACGTTATCGCCAGGATAGCTGGTATGCTGCCGTCTTCTGACAAGAAGTGTCGAGCGTGGGT
>Chuck-wills-widow
AGTAAACATCTCGAAAAACGTGGGTAGGCATGGACACGAGTGATACCATCCATATGTCAACTCATACTAGTTGCAGGTTAACAATACTACATGACCTATTACCGATCGAGATATGACAGAATAATTAATCTAAGTCCCAACAATGTCTCCAAGACAGGTGGTACGGTGCCGTCTTCCGACAAGAAGTGACGAACGTGAGT
>Killdeer
AGTAAGCATCCCGAAAAACGTGCGTAGGCATAGACACGAGTGACACCAGCAATATGTCAACTCATACTCGTAGTAAGATAACAATACTACATGACTCATTACCGATAGAGTTATGGAAGAATAAATAATCTAAGTCCCAACATTATCACCAGGGTCGGTGGTATGGTACCGTCTTCCGACAAGAAGTGTCGAACTTGGGC
>Turkey
AGGGAACATCTCGAAAAACGTGCGTAAGTGTAGGAATGGGTGATACCATCAATATGCCAACTCATACTTGGAGCGAGATAAACATGCTACTTGATTTATCGCCGGTAAAAATATGAGAGAATAAACAATCTAAGTCCCAACATTGTTGCCAGGTTAGGTGGTATGGTGCCGTATTCCGACAGGAAATGTCGAATGTGATC
>Great-crested-grebe
GGTAAGCTCCCCGAAAAACGTACGTAGGCATAGACACAAGTGATAACATCAATATGTCAACTCATACTCGTAGCAAGATAACAATACTACATGACCTATTACCGATAGAGATATGGAGGAATAAATAATCTAAGTCCCAACATTATCACCAGGACAGGAGGTATGGTGCCGTCTACCGACAGGAAGTGTCGGACGTGAGT
>Chimney-swift
AGTAAAGATCTCGAAAAACGTGCGTAGGCACAGACACGAGTGATACCATCACTATGTCGAGCCAAACTCGTTGCGAGATATCAATACTACGTGACCTATTACTGATAGAGATATGAAAGACTAAATAATCTGAGTCCCAAAATTATCACCAGGACAAGTGGTATGGTGCCATCTTCAGAGAAAAAGTGTCGAACGTGAGC
>Zebra-finch
AGTAAACATCTCGAAAAAAGTGTTTAGGTATAGACACTCGTGGTACCATCAATATGTCAACTCATACCCGTAGCGAGATAACAACACTATATGAACTATTACGAATGGTGGCGTGAAAGAATAAATAACCTAAGTCCCAACATTATCATTAGGATAGGTGGTATGGGGCCGTTTTCCGACAGTAGGGGTCGAACCTGAAC
>Annas-hummingbird
GGTAAGCATTTTGAAAAACGTGCGTAGGCATAGAAACGAGTGATACCATCAATATGTCATATCATACTCGTTGCAAGACATCAATACAACATGGCCTATCACCGATGGAGAGATGAAAGACTCAACAATCTAAGTCCCAACATTCTCACGAGGACAAGAGTTATGGTGCTATCTTCCGACAGGAAGTGCCAAACGTGAGC
>Turkey-vulture
AGTAAGCATCTCGAAAAGCGTGCGTAGGCATAGACACGAGTGATACCATCAATATGTCAACTCATACTCGTAGCAAAATAACAATACTACATGACCTATTACCGATAGAAATATGGAAGAATAAATAATCTAAGTCCCAACGTTATCACCAGGATAGGTGGTATGCTGCCGACTTCCGACAAGAAGTGTCGAACGTGAGC
>Grey-crowned-crane
ATTAAGCATCTCGAAAAACGGGCGTAGGCATAGACACGAGTGATACCATCAATATGTCAACCCACATTCGTAGCAAGATATCAATACTACATGATCTATTACCGACAGAGATATAGAAGAATAAATAATCTAAGTCCCGACATTATCACCAGGATAGGTGGCATGGTACCGTCTTTCGACAAGAACTGTCGAACGTGAGC
>Cuckoo-roller
AGTAAGCATCTCGAAAAACGTGCCTAGAAGTAGACAAGGGTGATATCATCAATATGTCAACTCATATTCGTAGCAAGATAACAATACTACACGACCTATTACCGATAGAGATATGGAAGAATAAATAATCTAAGTCCCAACATTGTCACCAGGATAGGTGGTATGGTGCCGTCTTCGGACAAGAAGTGGCAGATGTGAGC
>Common-cuckoo
GGTAAACATCTCGAAAAACGTGCGTAGGCAGAGACGCGAGTGATGCCATCGATATGTCAACTCCTACTTGTAGCTGGATAACAATATTACATGACCTATTACCGATCGAGATGTGAAAGAGTAAATAATCTAAGTCGCAGCATAATCACCAGGATAGGTGGTGTGGTACCGTCTTCCCACAAGAAGTGTCGAACGTGAAC
