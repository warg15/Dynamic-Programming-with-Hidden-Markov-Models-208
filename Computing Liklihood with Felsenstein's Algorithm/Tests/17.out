-3556.2462195221465
